modules/pricer/src/main/java/com/opengamma/strata/pricer/calibration/CalibrationValue.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (C) 2015 - present by OpenGamma Inc. and the OpenGamma group of companies
+ * 
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.pricer.calibration;
+
+import java.util.List;
+
+import com.opengamma.analytics.math.function.Function1D;
+import com.opengamma.analytics.math.matrix.DoubleMatrix1D;
+import com.opengamma.strata.finance.Trade;
+import com.opengamma.strata.pricer.rate.RatesProvider;
+
+/**
+ * Provides the calibration value.
```

Comment:
Can this be expanded? It's not immediately obvious to me what a calibration value is.